docs/concepts/identifier.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Identifier
+headline: Identifier
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Identifier</h2> 
+
+Spine framework supports the following types of identifiers:
+
+* Integer,
+* Long,
+* String,
+* A generated Java class implementing the `Message`.
+
+We recommend you to select Identifiers for your IDs and encourage using message-based IDs to avoid possible performance issues. 
```

Comment:
Please replace `to avoid possible performance issues.` with `to make your API strongly pronounced and type-safe.`